views/layouts/HeroLayout.css

```diff
@@ -0,0 +1,12 @@
+.HeroLayout_container {
+  display: flex;
+  flex-direction: column;
+  height: 897px;
+}
+.HeroLayout_main {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  flex-direction: row;
+  height: 100%;
```

Comment:
should this be a minimum height?

Reply:
[9d7c4dc](https://github.com/AMP-SCZ/dpdash/pull/468/commits/9d7c4dc05024fa07956aaa578a84f844f590db07)